crates/subspace-core-primitives/src/lib.rs

```diff
@@ -127,6 +131,9 @@ pub type SolutionRange = u64;
 /// The closer solution's tag is to the target, the heavier it is.
 pub type BlockWeight = u128;
 
+/// Block hash (the bytes from H256)
+pub type BlockHash = [u8; 32];
```

Comment:
Why is this necessary? Only for `injected_block_hash`? And if it is necessary isn't there a struct for it already?

Reply:
We only have substrate Block::Hash afaik. That is abstract, the actual bytes need to be extracted out of it (the H256 bytes).